File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/scale/QimangxingActivity.java
package com.github.kongpf8848.animation.activity.tween.scale;

import android.os.Bundle;
import android.view.animation.Animation;
import android.view.animation.ScaleAnimation;
import android.widget.ImageView;

import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.BaseActivity;

import butterknife.BindView;

/**
 * Created by jack on 2016/8/7.
 */
public class QimangxingActivity extends BaseActivity {


    @BindView(R.id.anim_iv)
    ImageView anim_iv;

    @Override
    protected int getLayoutId() {
        return R.layout.activity_scale_qimangxing;
    }

    @Override
    protected void onCreateEnd(Bundle savedInstanceState){
        ScaleAnimation scaleAnimation = new ScaleAnimation(1.0F, 1.2F, 1.0F, 1.2F, Animation.RELATIVE_TO_SELF, 0.5F, Animation.RELATIVE_TO_SELF, 0.5F);
        scaleAnimation.setDuration(2000);
        scaleAnimation.setFillAfter(true);
        anim_iv.startAnimation(scaleAnimation);
    }
}

File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/scale/QimangxingActivity.kt
package com.github.kongpf8848.animation.activity.tween.scale

import android.os.Bundle
import android.view.View
import android.view.animation.Animation
import android.view.animation.ScaleAnimation
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.base.BaseActivity

class QimangxingActivity : BaseActivity() {

    lateinit var anim_iv: View
    override fun getLayoutId(): Int {
        return R.layout.activity_qimangxing
    }

    override fun fitsSystemWindows(): Boolean {
        return false
    }

    override fun statusBarColor(): Int {
        return R.color.transparent
    }

    override fun statusBarDarkFont(): Boolean {
        return false
    }

    override fun onCreateEnd(savedInstanceState: Bundle?) {
        anim_iv=findViewById(R.id.anim_iv)
        val scaleAnimation = ScaleAnimation(1.0f, 1.2f, 1.0f, 1.2f, Animation.RELATIVE_TO_SELF, 0.5f, Animation.RELATIVE_TO_SELF, 0.5f)
        scaleAnimation.duration = 2000L
        scaleAnimation.fillAfter = true
        anim_iv.startAnimation(scaleAnimation)
    }
}